dist/stampit.js

```diff
@@ -710,9 +738,13 @@ isStamp = function isStamp(obj) {
  * @return {Function}             A composable stampit factory
  *                                (aka stamp).
  */
-convertConstructor = function convertConstructor(Constructor) {
-  return stampit().methods(Constructor.prototype).enclose(Constructor);
-};
+function convertConstructor(Constructor) {
```

Comment:
Couldn't this whole thing be rewritten with no change in instance behavior as:

```js
  return stampit().state(Constructor.prototype).enclose(Constructor);
```

?